duckdns/rootfs/etc/s6-overlay/s6-rc.d/duckdns/run

```diff
@@ -27,20 +27,23 @@ function le_renew() {
     # Prepare domain for Let's Encrypt
     for domain in ${domains}; do
         for alias in $(jq --raw-output --exit-status "[.aliases[]|{(.alias):.domain}]|add.\"${domain}\" | select(. != null)" /data/options.json) ; do
-            aliases="${aliases} ${alias}"
+            aliases="${aliases}"$'\n'"${alias}"
```

Comment:
Is there a particular reason you are using newlines as delimiter here? It feels it over-complicates things a bit. 

Can you try to use bash arrays?